modules/grub/nixos.nix

```diff
@@ -57,15 +58,13 @@ in
       ];
     })
   ];
-  options.stylix.targets.grub = with config.lib.stylix; {
+  options.stylix.targets.grub = {
     enable = mkEnableTarget "GRUB" true;
     useWallpaper = mkEnableWallpaper "GRUB" false;
   };
 
   config.boot.loader.grub =
     with config.lib.stylix.colors.withHashtag;
-    with config.lib.stylix;
-    with config.stylix.fonts;
```

Comment:
This was used at the end of the file, in `${mkGrubFont sansSerif}`. That needs to be changed to `${mkGrubFont fonts.sansSerif}`.